mne/viz/_brain/_brain.py

```diff
@@ -1030,6 +1034,96 @@ def update_auto_scaling(self, restore=False):
+        through keyword arguments that depend on the format. For available
+        formats and corresponding parameters see the imageio documentation:
+        http://imageio.readthedocs.io/en/latest/formats.html#multiple-images
+
+        .. Warning::
+            This method assumes that time is specified in seconds when adding
+            data. If time is specified in milliseconds this will result in
+            movies 1000 times longer than expected.
+
+        Parameters
+        ----------
+        filename : str
+            Path at which to save the movie. The extension determines the
+            format (e.g., `'*.mov'`, `'*.gif'`, ...; see the :mod:`imageio`
+            documenttion for available formats).
+        time_dilation : float
+            Factor by which to stretch time (default 4). For example, an epoch
+            from -100 to 600 ms lasts 700 ms. With ``time_dilation=4`` this
+            would result in a 2.8 s long movie.
+        tmin : float
+            First time point to include (default: all data).
+        tmax : float
+            Last time point to include (default: all data).
+        framerate : float
+            Framerate of the movie (frames per second, default 24).
+        interpolation : str | None
+            Interpolation method (``scipy.interpolate.interp1d`` parameter,
+            one of 'linear' | 'nearest' | 'zero' | 'slinear' | 'quadratic' |
+            'cubic'). If None, it is set internally ('nearest').
+            Defaults to None.
+        **kwargs :
+            Specify additional options for :mod:`imageio`.
+        """
+        import imageio
+        from scipy.interpolate import interp1d
+        from math import floor
+
+        # find imageio FFMPEG parameters
+        if 'fps' not in kwargs:
+            kwargs['fps'] = framerate
+        if codec is not None:
+            kwargs['codec'] = codec
+        if bitrate is not None:
+            kwargs['bitrate'] = bitrate
+
+        if interpolation is None:
+            interpolation = self.interp_kind
+
+        # find tmin
+        if tmin is None:
+            tmin = self._times[0]
+        elif tmin < self._times[0]:
+            raise ValueError("tmin=%r is smaller than the first time point "
+                             "(%r)" % (tmin, self._times[0]))
+
+        # find indexes at which to create frames
+        if tmax is None:
+            tmax = self._times[-1]
+        elif tmax > self._times[-1]:
+            raise ValueError("tmax=%r is greater than the latest time point "
+                             "(%r)" % (tmax, self._times[-1]))
+        n_frames = floor((tmax - tmin) * time_dilation * framerate)
+        times = np.arange(n_frames, dtype=float)
+        times /= framerate * time_dilation
+        times += tmin
+        interp_func = interp1d(self._times,
+                               np.arange(self._n_times),
+                               interpolation)
+        time_idx = interp_func(times)
```

Comment:
Do not `interp1d` over the times using the `interpolation`, this should always be linear (fractional time points). What needs to be interpolated using the `interpolation` order/mode are the actual data.